README.md

```diff
@@ -185,14 +168,34 @@ Sample screenshot:
 
 ![](images/sensor.[zone_name].png?raw=true)
 
+#### Services
+
+| Service | Description|
+| --- | --- |
+|`Smart Irrigation: calculate_zone`|Triggers the calculation of one specific zone.|
+|`Smart Irrigation: calculate_all_zones`|Triggers the calculation of all automatic zones. Use only if you disabled automatic refresh in the options.|
+|`Smart Irrigation: update_zone`|Updates one specific zone with weather data|
+|`Smart Irrigation: update_all_zones`|Updates all automatic zones with weather data|
+|`Smart Irrigation: reset_bucket`|Resets one specific bucket.|
+|`Smart Irrigation: reset_all_buckets`|Resets all automatic buckets.|
+|`Smart Irrigation: set_bucket`|Sets a specific bucket to to a specific `value`.|
+|`Smart Irrigation: set_all_buckets`|Sets all automatic buckets to a specific `value`.|
```

Comment:
all buckets are set, not just the buckets on the automatic zones

Reply:
learning by translation :-)